File: onebot/src/main/java/cn/evole/onebot/sdk/response/contact/FriendInfoResp.java
package cn.evole.onebot.sdk.response.contact;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Created on 2022/7/8.
 *
 * @author cnlimiter
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class FriendInfoResp {

    @SerializedName("user_id")
    public long userId;

    @SerializedName(value = "nickname", alternate = {"user_name"})
    public String nickname;

    @SerializedName(value = "remark", alternate = {"user_remark"})
    public String remark;

}

File: onebot/src/main/kotlin/sdk/response/contact/FriendInfoResp.kt
package cn.evolvefield.onebot.sdk.response.contact

import com.google.gson.annotations.SerializedName
import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor

@Data
@AllArgsConstructor
@NoArgsConstructor
class FriendInfoResp  {
    @SerializedName("user_id")
    var userId = 0L
    @SerializedName(value = "nickname", alternate = ["user_name"])
    var nickname = ""
    @SerializedName(value = "remark", alternate = ["user_remark"])
    var remark = ""
}
